src/styles/main.scss

```diff
@@ -1,3 +1,27 @@
-body {
+
+@import "./block/placeholders.scss";
+@import "./block/container.scss";
+@import "./block/header.scss";
+@import "./block/card.scss";
+@import "./block/stars.scss";
+
+ul,
+li {
   margin: 0;
+  padding: 0;
+}
```

Comment:
add reset and normalize css files for such styles like `margin: 0` `text-decoration: none;` `font-family: ;` and atc.